main.py

```diff
@@ -281,6 +287,18 @@ async def scrape_channel_media(channel: TextChannel) -> List[Tuple[Member, str,
     return channel_media_attachments
 
 
+def pick_random_emoji() -> str:
+    """Picks a random emoji from the loaded emoji list"""
+
+    # Choose a random emoji
+    encoded_random_emoji = random.choice(emoji_list)
+
+    # Decode the emoji from the unicode characters
+    decoded_random_emoji = encoded_random_emoji.encode("Latin1").decode()
+
+    return decoded_random_emoji
+
+
```

Comment:
Seems like there's an extra line here? Probably want to rebase and lint before merging

Reply:
Ah! That's actually PEP8 convention - top-level functions and classes should be surrounded by two blank lines: https://www.python.org/dev/peps/pep-0008/#blank-lines

The linting tools will instead enforce that.